examples/webpack/shims-vue.d.ts

```diff
@@ -0,0 +1 @@
+declare module '*.vue'
```

Comment:
This should probably be

```suggestion
/* eslint-disable */
declare module '*.vue' {
  import type { DefineComponent } from 'vue'
  const component: DefineComponent<{}, {}, any>
  export default component
}
```